src/components/App/CodeSplit/LazyLoaded.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Text } from 'ui';
+import { FadeTransform } from 'react-animation-components';
+
+const LazyLoaded = () =>
+    console.log('Lazy loaded 👏') || (
```

Comment:
While this lazy loads the component are we still code splitting the dependency? The purpose of this example in the boiler was to show how we could code split heavy modules like `GSAP` so theyre only loaded when needed and then once loaded, allow the component to render. While code splitting components is also useful, I feel like its more common to code split a large module like `GSAP` or `ThreeJS`

Reply:
Yes, the bundle that React dynamically loads breaks out and includes the imported dependencies as well. If you're seeing `react-animation-components` included in the client bundle, you're right. Magic imports are the culprit there 👎 You can test this by removing `ImgPreload` and running a build again. The animation library is only then included in the code split bundle.